doc/classes/ColorPicker.xml

```diff
@@ -90,6 +90,11 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="choose_color_close_requested">
+			<description>
+				Emitted when the user requests closing the [ColorPicker] after choosing a color. This occurs when the [ColorPicker] is part of a [ColorPickerButton] and the user holds [kbd]Ctrl[/kbd] when confirming a color choice by left-clicking or pressing [kbd]Enter[/kbd] in a text field. This behavior doesn't occur with standalone [ColorPicker]s.
```

Comment:
The last sentence is misleading. The signal is still emitted in standalone ColorPickers, it's just not used.